src/AccessibilityInsights.SharedUx/ActionViews/ReturnA11yElementsView.xaml.cs

```diff
@@ -56,15 +57,15 @@ public static bool HighlightVisibility
             }
             set
             {
-                var ha = HighlightOverlayAction.GetDefaultInstance();
-                var sha = HighlightAction.GetDefaultInstance();
+                var ha = ClearOverlayDriver.GetDefaultInstance();
```

Comment:
>ha [](start = 20, length = 2)

These local variable names seem kind of weird for the renamed types

Reply:
good catch, I did an auto-rename and will update these types